server/src/main/java/org/opensearch/index/LogByteSizeMergePolicyProvider.java

```diff
@@ -50,7 +50,7 @@ public class LogByteSizeMergePolicyProvider implements MergePolicyProvider {
     private final Logger logger;
     private final boolean mergesEnabled;
 
-    public static final ByteSizeValue DEFAULT_MIN_MERGE = new ByteSizeValue(2, ByteSizeUnit.MB);
+    public static final ByteSizeValue DEFAULT_MIN_MERGE = new ByteSizeValue(16, ByteSizeUnit.MB);
```

Comment:
Can you please add a comment just above this line explaining why we're not using the Lucene 10.x defaults?